testsuite/features/reposync/srv_sync_channels.feature

```diff
@@ -82,6 +82,16 @@ Feature: Be able to list available channels and enable them
     And I should get "[I] RES-AS-8-Updates for x86_64 SUSE Linux Enterprise Server with Expanded Support 8 x86_64 [res-as-8-updates-x86_64]"
     And I should get "[I] RES-CB-8-Updates for x86_64 SUSE Linux Enterprise Server with Expanded Support 8 x86_64 [res-cb-8-updates-x86_64]"
 
+@susemanager
+  Scenario: Enable SLES15 SP4 SUMA client tools for creating bootstrap repositories
+    When I execute mgr-sync "add channel sle-manager-tools15-pool-x86_64-sp4"
```

Comment:
(i never liked this direct bash coding inside of feature files)

Reply:
Yes. But if we are going to improve this, we will have to introduce several steps for replacing the current one. Furthermore we agreed to do more using the API/command line so moving this to the webUI makes no sense.

Reply:
yes, i was thinking of using the command as currently... just using English to state what we want instead of throwing an almost command line into the cucumber text